### ISSUE TITLE
hide the audio slide

### ISSUE THREAD

**Issue by guaka** [guaka/radio#50]
It's quite pointless for a stream
(And what's the exact term for this thing?)

![screen shot 2015-07-27 at 12 08 43](https://cloud.githubusercontent.com/assets/54728/8903424/46d921b4-3458-11e5-8553-9a076f2b43ac.png)

**Comment by guaka** [comment]
asked at http://stackoverflow.com/q/31819425/1245190

**Comment by simison** [comment]
You don't have much control over standard HTML5 audio element, but there are loads of small and bigger libraries that implement everything imaginable, for instance:

http://buzz.jaysalvat.com/
http://kolber.github.io/audiojs/
http://jplayer.org/
http://mediaelementjs.com/

You pretty much need just a volume control. How about just [hiding controls](http://www.w3schools.com/tags/av_prop_controls.asp) for that audio element and  implementing a separate `<input type="range">` to control volume? (or some JS slider solution)

**Comment by guaka** [comment]
Ah! Good idea, I didn't really think of these.

I also thought volume control wasn't needed, but @chmac requested it #42 and I have actually used it a couple of times, since I implemented it.

**Comment by simison** [comment]
Yep, volume control is pretty much the only one you really need.
